shard.yml

```diff
@@ -9,8 +9,10 @@ dependencies:
   active-model:
     github: spider-gazelle/active-model
     version: ~> 2
-  future:
-    github: crystal-community/future.cr
+  async:
+    # github: protoncr/async
+    # branch: main
+    path: ../../personal/async
```

Comment:
Could you change this to a pinned version?

Reply:
Whoops, will do

Reply:
Done!